cfme/configure/access_control.py

```diff
@@ -1213,6 +1214,7 @@ def is_displayed(self):
 class AllTenantView(ConfigurationView):
     """ All Tenants View """
     toolbar = View.nested(AccessControlToolbar)
+    table = Table('//*[@id="records_div"]/table')
```

Comment:
**REQUIRED**
This should use [entities](https://github.com/ManageIQ/integration_tests/blob/master/widgetastic_manageiq/__init__.py#L2630)